recipes/intx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,10 @@
+// intx: extended precision integer library.
+// Copyright 2019 Pawel Bylica.
+// Licensed under the Apache License, Version 2.0.
+
```

Comment:
Including a copyright notice on a single line of code seems a bit excessive to me.